core/lib/contract_verifier/src/compilers/mod.rs

```diff
@@ -31,11 +75,17 @@ fn parse_standard_json_output(
         let errors = errors.as_array().unwrap().clone();
         if errors
             .iter()
-            .any(|err| err["severity"].as_str().unwrap() == "error")
+            .any(|err| err["severity"].as_str() == Some("error"))
         {
+            // FIXME: collected messages contain warnings as well; is this intentional?
```

Comment:
Subj. On the one hand, it sort of makes sense to include warnings from the DevEx perspective, but the fact that they are not distinguished from errors (both are stored as message strings) is somewhat confusing.

Reply:
I think it's OK for now to not spend extra time on it. I don't think it was intentional, but I also don't think it's a big issue.